common.tf

```diff
@@ -175,6 +177,7 @@ module "ingest_step_function_policy" {
     ingest_upsert_archive_folders_lambda_name = local.ingest_upsert_archive_folders_lambda_name
     ingest_asset_opex_creator_lambda_name     = local.ingest_asset_opex_creator_lambda_name
     ingest_folder_opex_creator_lambda_name    = local.ingest_folder_opex_creator_lambda_name
+    ingest_start_workflow_lambda_name         = local.ingest_start_workflow_lambda_name
```

Comment:
You need to add a line into `ingest_step_function_policy.json.tpl` to use this variable.